operations/power_management/Prepare_the_System_for_Power_Off.md

```diff
@@ -249,6 +249,8 @@ HPE Cray EX System Admin Toolkit (SAT) product stream documentation (`S-8031`) f
 
 1. (`ncn-mw#`) Check for running sessions.
 
+   > *Note:* This step may take a longer time if there are many BOS sessions running. It is recommended to keep the sessions count minimal to reduce the overall time taken.
```

Comment:
Hey, @rustydb and @nrockershousen, I was actually reviewing this and looking at the underlying cause of the slowness. A couple comments:

1. The edit to add the "running" qualifier on the end of "many BOS sessions" is incorrect. They do not have to be running. They just have to exist on the system because the code in `sat bootsys` is checking all existing sessions to see if they are running.
2. This note should not be needed in CSM 1.6. In that version, we are using BOS v2, which does not have the slowness issue because it checks session status in a more efficient way. I'd suggest that we only make this change in documentation for CSM v1.3 (the last version for which BOS v1 was the default). Adding it unnecessarily in CSM 1.6 just adds more confusion to an already confusing shutdown process.

Can we revert here and have @shivaprasad-metimath make a change like this only to CSM 1.3 docs?